solutions/swb-reference/integration-tests/tests/isolated/users/listUsers.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import ClientSession from '../../../support/clientSession';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('list users negative tests', () => {
+  const setup: Setup = Setup.getSetup();
+  let adminSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('with invalid paginationToken', () => {
```

Comment:
should include a test to throw invalid parameters to ensure we are validating request inputs

Reply:
Got it! Added a few more to validate `pageSize` and `paginationToken`